pyproject.toml

```diff
@@ -0,0 +1,58 @@
+[tool.poetry]
+name = "crdch-model"
+version = "1.1.6"
+description = "CRDC-H model in LinkML, developed by the Center for Cancer Data Harmonization (CCDH). This Python package contains the dataclasses necessary to build and operate over data objects according to the CRDC-H model."
+readme = "CRDCH_MODEL.md"
+authors = [
+  "Mark A. Miller <MAM@lbl.gov>",
+  "Gaurav Vaidya <gaurav@ggvaidya.com>"
+]
+license = "MIT"
+packages = [
+  { include = "crdch_model.py", from = "crdch_model" }
+]
+include = [
+  "README.md",
+  "model/*",
+  "crdch_model/*"
+]
+keywords = [
+  'linkml',
+  'crdc',
+  'cancer',
+  'biomedical data'
+]
+classifiers = [
+  'Topic :: Scientific/Engineering :: Bio-Informatics',
+  'Intended Audience :: Healthcare Industry',
+  'Development Status :: 4 - Beta',
+  'Intended Audience :: Developers',
+  'Intended Audience :: Science/Research',
+  'Intended Audience :: Information Technology',
+  'License :: OSI Approved :: MIT License',
+  'Programming Language :: Python :: 3 :: Only',
+  'Programming Language :: Python :: 3.7',
+  'Programming Language :: Python :: 3.8',
+  'Programming Language :: Python :: 3.9'
+]
+
+[tool.poetry.urls]
+"CCDH Website" = "https://ccdh.cancer.gov"
+"CCDH Model" = "https://cancerdhc.github.io/ccdhmodel/"
+"GitHub Repository" = "https://github.com/cancerDHC/ccdhmodel/"
+"Bug Tracker" = "https://github.com/cancerDHC/ccdhmodel/issues"
+
+[tool.poetry.dependencies]
+python = ">=3.7.1,<3.10" # Because of https://github.com/linkml/linkml/issues/508
+linkml-runtime = {git = "https://github.com/linkml/linkml-runtime", rev = "main"}
+
+[tool.poetry.dev-dependencies]
+tox = "^3.24.4"
+linkml = {git = "https://github.com/cancerDHC/linkml.git", rev = "ccdh-dev"}
+mkdocs-material = "^7.3.6"
+mike = "^1.1.2"
+sheet2linkml = "^1.3.0"
+
```

Comment:
Do we want black on [tool.poetry.dev-dependencies]?

Reply:
Hm, are you asking that question because in one of our other repositories we mandate the formatting of code with black and you think it should be included as a regular tool dependency as opposed to a dev dependency?

Because otherwise tools like black, flake8, pytest, coverage, etc. should normally be included as dev dependencies itself, since we don't need those tools per se, to build and use the application successfully.

Reply:
Oh, I see. We don't build a package on this example-data repo, so we don't need black, pytest, coverage... Thanks, @sujaypatil96.

Reply:
@jooho-lee-kim This is the ccdhmodel repo, not the example-data repo -- but you're right, we do have a _little_ amount of Python code (in the `/tests` directory) that we can enforce code style on. I'll do in another PR (#165). The only other Python file here is the autogenerated Python data classes file (`/crdch_model/crdch_model.py`), but there's no point in enforcing code style here.

We also have some tests here, but they use unittest rather than pytest, so we don't need any additional packages for that. There's a GitHub Action for testing that (via `make test`).

It might actually be quite interesting to see what the code coverage is for the generated Python Data Classes! I'll create an issue for that at https://github.com/cancerDHC/ccdhmodel/issues/166

Thanks for your feedback, Jooho and @sujaypatil96!

Reply:
@gaurav right.. this is ccdhmodel repo. I realized that later I posted the comment. Anyway, I was confused with the purpose of using tools like black, flake8, and pytest.. Thank you for explaining.